contingency/contingency-dsl/src/main/java/com/powsybl/contingency/dsl/ExtendableDslExtension.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package com.powsybl.contingency.dsl;
+
+import com.powsybl.commons.extensions.Extendable;
+import com.powsybl.commons.extensions.Extension;
+import groovy.lang.Binding;
+import groovy.lang.MetaClass;
+
+import java.util.List;
+
+/**
+ * @author Paul Bui-Quang <paul.buiquang at rte-france.com>
+ */
+public interface ExtendableDslExtension<E extends Extendable<E>> {
+
+    Class<E> getExtendableClass();
+
+    void addToSpec(MetaClass extSpecMetaClass, List<Extension<E>> contingencyExtensions, Binding binding);
+}
```

Comment:
Is it really necessary to make this interface generic? I think if we make it generic, this interface is misplaced and should be moved into powsybl-dsl module.

Reply:
at first i thought this would be usefull for other extendable objects, and i tried to put it in common or dsl, then i stepped back.
I will put it in the dsl package though unused really since I extend it with specific ContingencyDslExtension now. Should we let it for future use or not ?